developer/frameworks/articles/configuration/11-dynamically-populating-select-fields.markdown

```diff
@@ -0,0 +1,107 @@
+---
+header-id: dynamically-populating-select-list-fields-in-the-configuration-ui
+---
+
+# Dynamically Populating Select List Fields in the Configuration UI
+
+[TOC levels=1-4]
+
+Configuration options can be presented in a select list, directly entering each
+label and value in the `@Meta.AD` annotation in the [Configuration
+interface](/docs/7-2/frameworks/-/knowledge_base/f/creating-a-configuration-interface).
+
+```java
+@Meta.AD(
+    deflt = "enabled-with-warning", name = "csv-export",
+    optionLabels = {"Enabled", "enabled-with-warning", "Disabled"},
```

Comment:
Usually a language keys is used for the label, so the value gets localized. And language keys are usually lower-cased.

Reply:
Good point. I copied this directly from our source code :scream: 
https://github.com/liferay/liferay-portal/blob/b51b7070c398d44462b282c69324b2bb62050dc4/modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/java/com/liferay/dynamic/data/lists/web/internal/configuration/DDLWebConfiguration.java#L38-L44

Reply:
Okay, it's fixed in master and on 7.2.x :+1:
https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-lists/dynamic-data-lists-web/src/main/java/com/liferay/dynamic/data/lists/web/internal/configuration/DDLWebConfiguration.java#L38-L44